index.js

```diff
@@ -81,19 +92,23 @@ let Rosnodejs = {
    * @return {Promise} resolved when connection to master is established
    */
   initNode(nodeName, options) {
+    options = options || {};
+    if (options.anonymous) {
+      nodeName = _anonymizeNodeName(nodeName);
+      console.log('anonymous node name ' + nodeName);
```

Comment:
Will people typically care that they see the node name when anonymizing though? `rospy` doesn't log anything. They can also ask for the name for the name of the node at any point.
```
console.log(rosnodejs.nh.getNodeName());
```